src/goCheck.ts

```diff
@@ -58,6 +58,14 @@ export function check(fileUri: vscode.Uri, goConfig: vscode.WorkspaceConfigurati
 	let runningToolsPromises = [];
 	let cwd = path.dirname(fileUri.fsPath);
 	let goRuntimePath = getBinPath('go');
+	const alternateLS = getAlternateLanguageServer(goConfig);
+	let languageServerFlags: string[] = goConfig.get('languageServerFlags');
+	if (!Array.isArray(languageServerFlags)) {
+		languageServerFlags = [];
+	}
+	const disableBuild = alternateLS === 'gopls' || (alternateLS === 'bingo'
+		&& languageServerFlags.indexOf('-diagnostics-style=none') === -1);
```

Comment:
kind of hard to read, can we break on multiple lines

Reply:
Done